src/ctx.h

```diff
@@ -51,6 +51,32 @@ namespace ispc {
 
 struct CFInfo;
 
+///////////////////////////////////////////////////////////////////////////
+/** AddressInfo is a helper class to work with pointers.
+    It keeps llvm pointer, llvm element type, and ISPC type.
+*/
+class AddressInfo {
+  public:
+    AddressInfo(){};
```

Comment:
Do we really want to do have default constructor without zero-initializing the members?

Reply:
We don't need it, agree.